addons/tutorial_04_split_tunneling3/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "api_version": "0.1",
+  "id": "tutorial_04_split_tunneling3",
+  "name": "Tutorial: Split tunneling",
+  "conditions": {
+    "enabled_features": ["splitTunnel"],
+    "min_client_version": "2.15.0"
```

Comment:
Do we have to change other add-ons too? Can we remove the version 2 of this addon? Or it's needed for 2.14?

Reply:
I think we still need them for older versions in case folks don't update. I can add `max_client_version` conditions to them though if they aren't already there.